andela/travel_tool_front
# #161050888 View deleted checklist items

#### What does this PR do?
The PR implements the view deleted checklist items feature.

#### Description of Task to be completed?
- View deleted checklist items. 

#### How should this be manually tested?
- A Travel Admin/Super Admin for a location
 - Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
      - Change into the project directory
      - Run the server `make start`
      - Login and navigate the trip planner page
      - Add a new checklist item using the add item button
      - Delete the created checklist item.
      - You should now see it under the deleted items header

#### Any background context you want to provide?
- You need to clone the [backend](https://github.com/andela/travel_tool_back.git) to test this functionality.

- **Add yourself as travel admin**
   -- Add your email address in the .env file on the backend as `DEFAULT_ADMIN=` and start/restart your server
   -- Grab your token from the browser after login and add the token as authorization on postman
   -- Send a put request to `http://localhost:5000/api/v1/user/admin`

#### What are the relevant pivotal tracker stories?
[#161050888](https://www.pivotaltracker.com/story/show/161050888)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-10-24 at 16 01 04" src="https://user-images.githubusercontent.com/12511966/47432159-2cbe1d80-d7a6-11e8-9713-93738d33bc40.png">


#### Questions:
N/A
